pages/api/questions/questionGenerator.ts

```diff
@@ -130,6 +130,7 @@ export const generateQuestionForSkill = (
         unitPriceModel: {
           total: randomTotal,
           numberOfObjects: randomNumberOfObjects,
+          answer: "",
```

Comment:
This looks like its not meant to be here. Maybe undo this? Or remove this change?